dev-docs/bidders/rixengine.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: bidder
+title: RixEngine
+description: Prebid RixEngine Bidder Adapter
+biddercode: rixengine
+tcfeu_supported: false
+coppa_supported: true
+schain_supported: false
+media_types: banner
+safeframes_ok: false
+deals_supported: false
+pbjs: false
+pbs: false
+pbs_app_supported: false
+prebid_member: true
+userIds: sharedId
+sidebarType: 0
+multiformat_supported: false
+---
+
+### Note
+
+The RixEngine Bidding adapter requires setup before beginning. Please contact the RixEngine team.
```

Comment:
@xiaochang can you provide a email address to contact?

Reply:
@muuki88 Please use yuanchang@algorix.co and austin@algorix.co. Thanks.

Reply:
Sorry, I phrased it badly. Can you add a contact address to the docs. Most Adapters have a separate address, e.g. prebid@algorix.io